docs/cog_guides/core.rst

```diff
@@ -1190,6 +1190,36 @@ mean some pages will exceed this limit.
 **Arguments:**
     - ``<limit>`` - The max amount of characters to show per page in the help message.
 
+
```

Comment:
Based on how the rest of the document looks, there should be one empty line less here.
```suggestion
```